langchain4j/src/test/java/dev/langchain4j/agent/tool/DefaultToolExecutorTest.java

```diff
@@ -296,4 +294,109 @@ public void should_not_execute_tool_with_null_execution_request() throws NoSuchM
                 .isThrownBy(() -> new DefaultToolExecutor(new TestTool(), (ToolExecutionRequest) null));
 
     }
-}
\ No newline at end of file
+
+    private static class PersonTool {
+
+        @Tool
+        public Person save( Person arg) {
+            assert arg != null;
+            return arg;
+        }
+
+        @Tool
+        public Person[] saveArray(Person[] arg) {
+            assert arg != null;
```

Comment:
doesn't look `assert` is used somewhere in project's tests. Can we consider using  Assertj library to be consistent?

Reply:
assert here is to fail the tool execution with error. Not a Junit.

Junit Asserts adding in Test methods.